Gemfile

```diff
@@ -29,12 +29,12 @@ gem "ffi", ">= 1.15.5", "< 1.17.0"
 # but our runtime dep is still 3.9+
 gem "rspec", ">= 3.10"
 
-group :omnibus do
-  gem "rb-readline"
-  gem "appbundler"
-  gem "ed25519" # ed25519 ssh key support done here as its a native gem we can't put in the gemspec
-  gem "bcrypt_pbkdf" # ed25519 ssh key support done here as its a native gem we can't put in the gemspec
-end
+# group :omnibus do
```

Comment:
should we just remove it instead of comments?

Reply:
We might need it in future.. this practice is also follow by Chef client team, so keeping them like this for now. Had removed them earlier but have reverted them to comments in latest commits